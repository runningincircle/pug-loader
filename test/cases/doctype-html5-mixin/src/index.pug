mixin img
  img(src='image1.png')

doctype html5
html
  body
    +img
    img(src='image2.png')