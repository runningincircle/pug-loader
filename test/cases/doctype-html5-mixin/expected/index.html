<!DOCTYPE html5><html><body><img src="image1.png"/><img src="image2.png"/></body></html>